pkg/cmd/pulumi/destroy.go

```diff
@@ -149,6 +150,34 @@ func newDestroyCmd() *cobra.Command {
 			if err != nil {
 				return result.FromError(err)
 			}
+
+			var protectedCount int
+			if excludeProtected {
```

Comment:
Separate function here? I get a bit hard to follow. Stylistic preference but I'm a fan of smaller named func.